### PR TITLE
feat: adopt hubble sdk on notebook login

### PR DESCRIPTION
**Pull request by bwanglzu**
The latest hubble sdk release brings finetuner sdk user experience improvement, see screenshot as follows:

5842EE6C-7556-4056-8E77-41F509D4E5E2

After success:

AE34DAB4-8CCB-4389-91C9-99B2A44BC116

---

- [ ] This PR references an open issue
- [x] I have added a line about this change to CHANGELOG

### CODE REVIEW

File: finetuner/finetuner.py [hunk 1]
```diff
@@ -27,6 +27,17 @@ def login(self):
         hubble.login()
         self._init_state()
 
+    def notebook_login(self, force: bool = False):
+        """Login to Hubble account, initialize a client object
```
Comment (alexcg1) [inline finetuner/finetuner.py:31]:
```suggestion
        """Log in to Hubble account, initialize a client object
```

File: finetuner/finetuner.py [hunk 2]
```diff
@@ -27,6 +27,17 @@ def login(self):
...
+        and create a default experiment.
+
+        :param force: If set to true, will overwrite token and re-login.
```
Comment (alexcg1) [inline finetuner/finetuner.py:34]:
```suggestion
        :param force: If set to true, overwrite token and re-login.
```

File: finetuner/finetuner.py [hunk 3]
```diff
@@ -27,6 +27,17 @@ def login(self):
...
+
+        Note: This works for notebook, colab..
```
Comment (alexcg1) [inline finetuner/finetuner.py:36]:
```suggestion
        Note: This works for Jupyter notebook, Google Colab..
```

File: docs/tasks/image-to-image.md [hunk 4]
```diff
@@ -43,7 +43,7 @@ From now on, all the action happens in the cloud!
 First you need to {ref}`login to Jina ecosystem <login-to-jina-ecosystem>`:
```
Comment (alexcg1) [inline docs/tasks/image-to-image.md:43]:
```suggestion
First, {ref}`log in to the Jina ecosystem <login-to-jina-ecosystem>`:
```

File: docs/get-started/how-it-works.md [hunk 5]
```diff
@@ -36,7 +36,7 @@ From an engineering perspective,
 we have hidden all the complexity of machine learning algorithms and resource configuration (such as GPUs).
 All you need to do is decide on your backbone model and prepare your training data.
 
-Once you logged into the Jina Ecosystem with {meth}`~finetuner.login()`, 
+Once you logged into the Jina Ecosystem with {meth}`~finetuner.login()` or `~finetuner.notebook_login()`,
```
Comment (alexcg1) [inline docs/get-started/how-it-works.md:39]:
```suggestion
Once you have logged in to the Jina Ecosystem with {meth}`~finetuner.login()` or `~finetuner.notebook_login()`, 
```

### Usage examples

- I will log in to foo
- I am logged in
- My login for foo is bar

God, I hate [grammar](https://english.stackexchange.com/questions/5302/log-in-to-or-log-into-or-login-to) :sweat_smile:

File: CHANGELOG.md [hunk 6]
```diff
@@ -31,6 +31,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Change CLIP fine-tuning example in the documentation ([#569](https://github.com/jina-ai/finetuner/pull/569))
 
+- Use latest hubble with `notebook_login` support ([#576](https://github.com/jina-ai/finetuner/pull/576))
```
Comment (alexcg1) [inline CHANGELOG.md:34]:
```suggestion
- Use latest Hubble with `notebook_login` support ([#576](https://github.com/jina-ai/finetuner/pull/576))
```

File: README.md [hunk 7]
```diff
@@ -129,7 +129,7 @@ The following code snippet describes how to fine-tune ResNet50 on [Totally Looks
 import finetuner
 from finetuner.callback import EvaluationCallback
 
-finetuner.login()
+finetuner.login()  # use finetuner.notebook_login() in jupyter notebook/google colab
```
Comment (alexcg1) [inline README.md:132]:
```suggestion
finetuner.login()  # use finetuner.notebook_login() in Jupyter notebook/Google Colab
```

Can you change all occurrences?

**Comment by bwanglzu** [comment]
@alexcg1 if you find this looks good please review again, otherwise will block us from merge

File: docs/walkthrough/integrate-with-jina.md [hunk 8]
```diff
@@ -76,7 +76,7 @@ You have three options:
 import finetuner
 from jina import Flow
 
-finetuner.login()
+finetuner.login()  # use finetuner.notebook_login() in Jupyter notebook or Google colab()
```
[the remark this answers is not in the document]
Comment (alexcg1) [reply docs/walkthrough/integrate-with-jina.md:79]: Also, should be **C**olab, not **c**olab

**Review by alexcg1 — CHANGES_REQUESTED** [review]

**Review by alexcg1 — APPROVED** [review]
LGTM :+1:

**Review by gmastrapas — APPROVED** [review]

**Comment by bwanglzu** [comment]
test failed blocked by : https://github.com/jina-ai/jina-hubble-sdk/pull/83

File: docs/tasks/text-to-image.md [hunk 9]
```diff
@@ -29,7 +29,7 @@ From now on, all the action happens in the cloud!
 First you need to {ref}`login to Jina ecosystem <login-to-jina-ecosystem>`:
 ```python
 import finetuner
-finetuner.login()
+finetuner.login()  # use finetuner.notebook_login() in Jupyter notebook or Google Colab
```
Comment (gmastrapas) [inline docs/tasks/text-to-image.md:32]: Can you update this tutorial with the correct da names? These are `fashion-train-data-clip` and `fashion-eval-data-clip`

**Review by gmastrapas — CHANGES_REQUESTED** [review]

**Review by LMMilliken — APPROVED** [review]

File: finetuner/finetuner.py [hunk 10]
```diff
@@ -27,6 +27,16 @@ def login(self):
         hubble.login()
         self._init_state()
 
+    def notebook_login(self, force: bool = False):
+        """Log in to Hubble account, initialize a client object
+        and create a default experiment.
+
+        :param force: If set to true, overwrite token and re-login.
```
Comment (guenthermi) [inline finetuner/finetuner.py:34]: Can you add the force parameter also to the login function. It also exists there.

**Review by guenthermi — APPROVED** [review]
LGTM, only added one comment to be fixed.

File: docs/tasks/text-to-image.md [hunk 11]
```diff
@@ -11,7 +11,7 @@ For each product the dataset contains a title and images of multiple variants of
 
 ## Data
 Our journey starts locally. We have to {ref}`prepare the data and push it to the cloud <create-training-data>` and Finetuner will be able to get the dataset by its name. For this example,
-we already prepared the data, and we'll provide the names of training and evaluation data (`clip-fashion-train-data` and `clip-fashion-eval-data`) directly to Finetuner.
+we already prepared the data, and we'll provide the names of training and evaluation data (`fashion-eval-train-clip` and `fashion-eval-data-clip`) directly to Finetuner.
```
Comment (gmastrapas) [inline docs/tasks/text-to-image.md:14]: the train data are `fashion-train-data-clip`. Update the snippet as well

**Review by gmastrapas — CHANGES_REQUESTED** [review]

**Review by gmastrapas — APPROVED** [review]

**Pull request merged**